be/src/util/mem_info.h

```diff
@@ -34,6 +34,8 @@ class MemInfo {
     // Initialize MemInfo.
     static void init();
 
+    static inline bool initialized() { return _s_initialized; }
```

Comment:
What is this for?

Reply:
Used in MemTracker::check_sys_mem_info to determine whether the check exceeds the upper limit during try consume.

(Actually, the problem of MemTracker try consume before MemInfo is initialized will only exist when TCMalloc Hook new/delete later)